rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -172,127 +169,159 @@ <h1 class="col-sm-10 col-md-11">{{ record.title }}</h1>
                     </div>
-                  {% endif %}
-                </div>
-                <div class="col-12 col-sm-9 d-flex flex-column">
-                  {% if current_user |can_access_item(item) %}
-                    {% with
-                      href_update=url_for('records/items.index', path=item.pid, document=record.pid),
-                      href_delete='/api/items/' + item.pid,
-                      json=item,
-                      next=url_for('invenio_records_ui.doc', viewcode=viewcode, recordType=recordType, pid_value=record.pid),
-                      message=_("Item cannot be deleted: there are still transactions linked to this item.")
-                    %}
-                      {% include 'rero_ils/_editor_button_actions.html' %}
-                    {% endwith %}
-                  {% endif %}
-                </div>
-              </footer>
-            </section>
+                  </div>
+                {% endif %}
           {% endfor %}
+            </div>
+            <!-- Accordion card -->
+        </section>
         {% endif %}
-        {% endfor %}
-      </section>
     </section>
-  <footer class="pt-4 rero-ils-detailed-view-footer">
-    {%- set formats = export_formats(pid.pid_type) %}
-      {%- if formats %}
-        <h3 class="d-inline-block">{{ _('Export Formats') }}:</h3>
-        <ul class="list-inline d-inline-block">
-        {%- for slug, fmt in formats %}
-          <li>
-            <a href="{{ url_for('invenio_records_ui.doc_export', viewcode=viewcode, pid_value=pid.pid_value, format=slug, prettyprint=True) }}">
-              {{ fmt.title }}
-            </a>
-          </li>
-        {%- endfor %}
-        </ul>
-      {%- endif %}
-  </footer>
-  {%- endblock record_body %}
+    <footer class="pt-4 rero-ils-detailed-view-footer">
+        {%- set formats = export_formats(pid.pid_type) %}
+          {%- if formats %}
+            <h3 class="d-inline-block">{{ _('Export Formats') }}:</h3>
+            <ul class="list-inline d-inline-block">
+            {%- for slug, fmt in formats %}
+              <li>
+                <a href="{{ url_for('invenio_records_ui.doc_export', viewcode=viewcode, pid_value=pid.pid_value, format=slug, prettyprint=True) }}">
+                  {{ fmt.title }}
+                </a>
+              </li>
+            {%- endfor %}
+            </ul>
+          {%- endif %}
+      </footer>
+
 {%- endblock body %}
 
 {%- block javascript %}
   {{ super() }}
   {% assets "rero_ils_documents_detailed_js" %}<script src="{{ ASSET_URL }}"></script>{% endassets %}
+  <script>
+    $(document).ready(function() {
+      $('.collapse').on('hidden.bs.collapse', function () {
+        var id = $(this).data('holdingId');
+        $('#' + id).find('.availability').show();
+      })
+
+      $('.collapse').on('shown.bs.collapse', function () {
+        var id = $(this).data('holdingId');
+        $('#' + id).find('.availability').hide();
+      })
+    });
+  </script>
```

Comment:
Put the javascript code in a separate file (e.g. documents/static/js/rero_ils/documents_jq.js) and add it to the bundles.py file